spring-boot-actuator/src/main/java/org/springframework/boot/actuate/trace/TraceProperties.java

```diff
@@ -48,6 +49,12 @@
 	 */
 	private Set<Include> include = new HashSet<Include>(DEFAULT_INCLUDES);
 
+	/**
+	 * Whether {@literal Cookie} and {@literal Set-Cookie} should be excluded from
+	 * request/response headers. Defaults to {@code false}.
+	 */
+	private Boolean excludeCookies = false;
```

Comment:
I think we should probably have an `Exclude` that pairs with the `Include` above. I don't know if having a bunch of well-known excludes is the way to go though. I have no better idea at this point.